core-bundle/src/Resources/contao/dca/tl_user.php

```diff
@@ -172,16 +173,16 @@
 		),
 		'language' => array
 		(
-			'default'                 => str_replace('-', '_', $GLOBALS['TL_LANGUAGE']),
+			'default'                 => LocaleUtil::formatAsLocale($GLOBALS['TL_LANGUAGE']),
 			'exclude'                 => true,
 			'filter'                  => true,
 			'inputType'               => 'select',
-			'eval'                    => array('rgxp'=>'locale', 'tl_class'=>'w50'),
+			'eval'                    => array('includeBlankOption'=>true, 'chosen'=>true, 'rgxp'=>'locale', 'tl_class'=>'w50'),
```

Comment:
Why do we have a blank option now? Is it valid to have a backend user without a language?

Reply:
The field is not mandatory, so I thought it should be the same as for members. But you're right, it should probably be mandatory instead